src/resources/3d/assetMappings3DApi.ts

```diff
@@ -19,18 +18,18 @@ export class AssetMappings3DAPI extends BaseResourceAPI<AssetMapping3D> {
    * const mappings3D = await client.assetMappings3D.list(3244265346345, 32423454353545);
    * ```
    */
-  public list(
+  public list = (
     modelId: CogniteInternalId,
     revisionId: CogniteInternalId,
     scope?: AssetMappings3DListFilter
-  ): CursorAndAsyncIterator<AssetMapping3D> {
```

Comment:
Do you want to remove all the explicit return types?

Reply:
they don't make sense in most places, because the functions we call are already typed
(at least I didn't notice any difference)
But anyway, what was the idea on having them on you side?

Reply:
Just to have explicit return values to give build errors when that isn't true anymore. Unsure how useful it is. So we can keep it as is for now (we have tests).

Reply:
agree here, will revert it back